etna/lib/etna/clients/magma/workflows/model_synchronization_workflow.rb

```diff
@@ -113,8 +115,37 @@ def ensure_model_attribute(model_name, attribute_name)
         # This method, and it's partner prepare_parent, should never call into any re-entrant or potentially
         # cyclical method, like ensure_model_tree.
         def ensure_model(model_name)
+          ensure_model_create(model_name)
+          ensure_model_identifier_configured(model_name)
+        end
+
+        def ensure_model_identifier_configured(model_name)
```

Comment:
Borrowing from @coleshaw 's work, allows synchronizing the properties of identifier attributes via updates after creation.